R/drive_rm.R

```diff
@@ -32,6 +33,5 @@ delete_one <- function(id) {
     endpoint = "drive.files.delete",
     params = list(fileId = id)
   )
-  response <- make_request(request)
-  process_response(response)
+  do_request(request)
```

Comment:
Can we standardize on `make_request()` + `process_response()`? I know it seems silly here, but I think it's better to be consistent. One of us will read this one day and wonder if there's a reason it's different.